custom_components/unfoldedcircle/manifest.json

```diff
@@ -10,7 +10,7 @@
   "iot_class": "local_push",
   "issue_tracker": "https://github.com/JackJPowell/hass-unfoldedcircle/issues",
   "requirements": [
-    "websockets==12.0",
+    "websockets>=12.0",
```

Comment:
Thanks guys. Based on a quick read of the change log, it doesn't look like this will be affected by v14, but we'll leave it like you have it until I can complete testing.